rtdata/languages/default

```diff
@@ -1286,9 +1285,6 @@ HISTORY_MSG_1061;Local - CIECAM Source absolute
 HISTORY_MSG_1062;Local - CIECAM Surround Source
 HISTORY_MSG_1063;Local - CIECAM Saturation
 HISTORY_MSG_1064;Local - CIECAM Chroma
-HISTORY_MSG_1062;Local - CIECAM Lightness
```

Comment:
Why do these get deleted? This exists https://github.com/Beep6581/RawTherapee/blob/0a554282054c061fb06422ebae1f218fec8003e2/rtengine/procevents.h#L1092 and is implemented here https://github.com/Beep6581/RawTherapee/blob/2924128311d7fa5165a477b83a84c57870c3799b/rtgui/locallabtools2.cc#L10001

Reply:
It's a duplicate key. We need to check which string is correct.

Reply:
The events match the strings that are still here so I think this is correct.
https://github.com/Beep6581/RawTherapee/blob/0a554282054c061fb06422ebae1f218fec8003e2/rtengine/procevents.h#L1091-L1093

Reply:
I looked at both, the code in "locallabtools2.cc", "procevents.h", "refreshmap.cc" and what appears in "history". There is duplication in "HISTORY_MSG"

The good code is
HISTORY_MSG_1061;Local - CIECAM Source absolute
HISTORY_MSG_1062;Local - CIECAM Surround Source
HISTORY_MSG_1063;Local - CIECAM Saturation
HISTORY_MSG_1064;Local - CIECAM Chroma
!HISTORY_MSG_1062;Local - CIECAM Lightness
!HISTORY_MSG_1063;Local - CIECAM Brightness
HISTORY_MSG_1064;Local - CIECAM threshold
HISTORY_MSG_1065;Local - CIECAM lightness J
HISTORY_MSG_1066;Local - CIECAM brightness
HISTORY_MSG_1067;Local - CIECAM Contrast J

So you have to delete the 2 lines with a !
!HISTORY_MSG_1062;Local - CIECAM Lightness
!HISTORY_MSG_1063;Local - CIECAM Brightness

jacques

Reply:
Duplicates were already deleted, nothing to do.